e2e/helpers.go

```diff
@@ -94,7 +99,18 @@ func UpdateScanConfigToStartNow(config *apitypes.ScanConfig) *apitypes.ScanConfi
 		},
 		Scheduled: &apitypes.RuntimeScheduleScanConfig{
 			CronLine:      config.Scheduled.CronLine,
-			OperationTime: to.Ptr(time.Now()),
+			OperationTime: to.Ptr(time.Now().Add(5 * time.Second)), // nolint:gomnd
 		},
 	}
 }
+
+// "database is locked" errors from SQLite should be ignored during the test
+// they occur frequently on Azure, but they are temporary and should not cause the test to fail.
+func skipDBLockedErr(err error) error {
```

Comment:
this should be removed in the future. lets have a ticket for this

Reply:
https://github.com/openclarity/vmclarity/issues/1654